Alarmierung/README.md

```diff
@@ -1,4 +1,7 @@
 # Alarmierung
+
+_Dieses Modul wird nicht mehr gepflegt. Die aktuelle Version ist hier zu finden: https://github.com/symcon/Alarmierung_
```

Comment:
Umfangreicherer Text bei allen Modulen:
Die aktuelle Version dieses Moduls ist nicht mehr in SymconMisc verfügbar. Die aktuelle Version finden Sie in dem seit Version 5.1 verfügbaren Module Store. Alternativ können Sie es über das Module Control installieren: <Link>